src/commands/mail/deliverybox/create.tsx

```diff
@@ -0,0 +1,153 @@
+import { Flags } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { projectFlags } from "../../../lib/resources/project/flags.js";
+import { ExecRenderBaseCommand } from "../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../rendering/process/process_flags.js";
+import { Text } from "ink";
+import { Success } from "../../../rendering/react/components/Success.js";
+import { ReactNode } from "react";
+import { ProcessRenderer } from "../../../rendering/process/process.js";
+import { FlagInput, OutputFlags } from "@oclif/core/lib/interfaces/parser.js";
+import crypto from "crypto";
+import { Value } from "../../../rendering/react/components/Value.js";
+
+type CreateResult = {
+  deliveryboxId: string;
+  generatedPassword: string | null;
+};
+
+export default class Create extends ExecRenderBaseCommand<
+  typeof Create,
+  CreateResult
+> {
+  static summary = "Create a new mail deliverybox";
+  static description = `\
+    This command can be used to create a new mail deliverybox in a project.
+  
+    When running this command with the --quiet flag, the output will contain the ID of the newly created deliverybox.
+    In addition, when run with --generated-password the output will be the ID of the newly created deliverybox, followed by a tab character and the generated password.`;
+  static flags = {
+    ...projectFlags,
+    ...processFlags,
+    description: Flags.string({
+      char: "d",
+      summary: "mail devliverybox description",
+      required: true,
+    }),
+    password: Flags.string({
+      summary: "deliverybox password",
+      description:
```

Comment:
Use the `exclusive` property (IIRC, refer to documentation) to configure that _either_ `--password` _or_ `--random-password` (or none of them) is supplied.